util/stmtsummary/evicted.go

```diff
@@ -190,3 +209,214 @@ func (seElement *stmtSummaryByDigestEvictedElement) toEvictedCountDatum() []type
 func (ssMap *stmtSummaryByDigestMap) ToEvictedCountDatum() [][]types.Datum {
 	return ssMap.other.ToEvictedCountDatum()
 }
+
+func (ssbde *stmtSummaryByDigestEvicted) toCurrentDatum() []types.Datum {
+	var seElement *stmtSummaryByDigestEvictedElement
+	ssbde.Lock()
+	defer ssbde.Unlock()
```

Comment:
The duration of locking can be shorter. `toDatum` doesn't need to lock `ssbde`. It may cause jitter when querying `statements_summary` in a high concurrency workload.
See how `stmtSummaryByDigest.toCurrentDatum` does.